packages/util-user-agent-node/src/index.ts

```diff
@@ -14,10 +14,7 @@ interface DefaultUserAgentOptions {
 /**
  * Collect metrics from runtime to put into user agent.
  */
-export const defaultUserAgent = ({
-  serviceId,
-  clientVersion,
-}: DefaultUserAgentOptions): Provider<UserAgent> => async () => {
+export const defaultUserAgent = ({ serviceId, clientVersion }: DefaultUserAgentOptions): Provider<UserAgent> => {
```

Comment:
I see the old code was `=> async () => {` (a function that returned an async function), meaning the old code would evaluate all of the common functions each time, including loading the app config.